src/blocks/element.py

```diff
@@ -33,3 +37,6 @@ def nested(self):
         """Could this element hold other elements inside, True for yes.
         """
         return True
+
+    def __eq__(self, value):
+        return isinstance(value, Element) and value.content() == self.content()
```

Comment:
equality of different type but same content().

Reply:
done